integrations/third_party/recorded_future_intelligence/pyproject.toml

```
@@ -1,6 +1,6 @@
 [project]
 name = "RecordedFutureIntelligence"
-version = "6.0"
+version = "6.1"
```

```suggestion
version = "7.0"
```
version need to be increased by 1.0